code/modules/projectiles/projectile/bullets.dm

```diff
@@ -35,10 +31,9 @@
 			M.AdjustDrowsy(10)
 			A.volume += 5 //Because we can
 
-/obj/item/projectile/bullet/weakbullet2
+/obj/item/projectile/bullet/rubber
 	name = "rubber bullet"
-	damage = 5
```

Comment:
all other rubber rounds deal a small amount of brute. feels weird to make this the exception